app/views/categories/_filter_labels.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class="filters-remove">
+  <div class="filters-remove__heading">
+    <div class="govuk-body filters-remove__heading-title govuk-!-margin-bottom-0">
+      Active filters
+    </div>
+  </div>
+
+  <ul class="filters-remove-tags">
+    <% @subcategories.select { |s| params[:subcategory_slugs]&.include?(s.slug) }.each do |subcategory| %>
```

Comment:
might use `it` here, but also I feel like this might be better set in the controller as `@removable_tags` or something like that.